transcribe.py

```diff
@@ -149,6 +149,60 @@ def transcript_file_name_from_video_file_name(video_file_name):
 
 	return transcript_job_name + ".json"
 
+def format_time(val):
+	secs = float(val)
+	hours = int(secs/3600)
+	secs = secs - (hours * 3600)
+	mins = int(secs/60)
+	secs = secs - (mins * 60)
+	m_secs = int(secs*1000)
+	secs = int(secs)
+	return pad_time(hours) + ':' + pad_time(mins) + ':' + pad_time(secs) + ',' + pad_time(m_secs, 3)
+
+def pad_time(val, l= 2):
+	return ('0'.join([''] * (l+1)) + str(val))[-l:]
+
+def convert_transcribe_to_srt(video_file):
+	if not transcript_exists_in_s3(video_file):
+		logger.info("Transcript for video file %s does not exist" % video_file)
+		return "Failed to convert to srt. Transcript does not exist."
+
+	logger.info("Conversion to srt started for video file: %s" % video_file)
+	with open(transcript_file_name_from_video_file_name(video_file), encoding='utf-8') as f:
+		raw = json.load(f)
+		items = raw['results']['items']
```

Comment:
if results are null or empty and items are null or empty, log the same and move on.